src/wam2layers/preprocessing/pressure_levels.py

```diff
@@ -0,0 +1,139 @@
+from wam2layers.preprocessing.utils import interpolate, sortby_ndarray
+
+
+import numpy as np
+import xarray as xr
+
+
+def extend_pressurelevels(
+    input: xr.Dataset,
```

Comment:
input vs surface is a bit confusing at first glance. Can we change it to e.g. 'input_3d', and 'input_surface'? Or pressure_level_variables vs surface_variables